tests/integration/README.md

```diff
@@ -133,58 +133,50 @@ $ python ./tests/integration/run.py run ./all/cases/...
+# Test the container image with test cases having the `macaron-docker-image` tag.
+$ python ./tests/integration/run.py run --include-tag macaron-docker-image ./all/cases/...
 ```
 
-The `--include-tag` flag can be specified multiple times. A selected test case must contain all tags specified with the `--include-tag` flag.
+We can do the same with `macaron-python-package` when we do integration tests for the Macaron Python package.
+
+The `--include-tag` flag can be specified multiple times. A selected test case must be tagged with at least a tag specified with any of the `--include-tag` flags.
 
 ```bash
-# Test the container image with test cases having the `docker` tag.
+# Test the container image with test cases having EITHER `tag-a` for `tag-b` tag.
 $ python ./tests/integration/run.py run --include-tag tag-a --include-tag tag-b ./all/cases/...
 ```
 
 There is also the `--exclude-tag` flag. A selected test case must also not contain any tag specified with the `--exclude-tag` flag.
 
 ```bash
-# Only run test cases not tagged with `npm`.
-$ python ./tests/integration/run.py run --exclude-tag npm ./all/cases/...
+# Only run test cases not tagged with `npm-registry-testcase`.
+$ python ./tests/integration/run.py run --exclude-tag npm-registry-testcase ./all/cases/...
 ```
 
-You can simply think of each `--include-tag`/`--exclude-tag` argument as adding an additional constraint that a selected test case must satisfy".
+You can simply think of each `--include-tag`/`--exclude-tag` argument as adding an additional constraint that a selected test case must satisfy.
 
-Some other use cases of this tagging feature in our current setup:
-- We can have test cases that **only** run for the container image. In the integration tests for the Macaron Python package, test cases marked with `docker-only` are not run. Note that `docker-only` shouldn't be used with `shared-docker-python`. Therefore our current test suite for the docker container must be run as separated commands:
-```bash
-# Run all test cases with `docker-only` tag.
-$ python ./tests/integration/run.py \
-      run \
-      --macaron scripts/release_scripts/run_macaron.sh \
-      --include-tag docker-only \
-      ./tests/integration/cases/...
-
-# Run all test cases with `shared-docker-python` tag.
-$ python ./tests/integration/run.py \
-      run \
-      --macaron scripts/release_scripts/run_macaron.sh \
-      --include-tag shared-docker-python \
-      ./tests/integration/cases/...
-```
-- Test cases marked with `skip` are not run.
-- Test cases marked with `npm-registry` are not run if the environment variable `NO_NPM` is set to `TRUE`. This only applies when you run the integration tests with:
+Instructions on how to tag a test case for our CI/CD pipeline:
+- If you want a test case to **only** run for the container image, using **only** `macaron-docker-image`.
+- If you want a test case to **only** run with the Macaron Python package **only** `macaron-python-package`.
```

Comment:
```suggestion
- If you want a test case to **only** run with the Macaron Python package, use **only** `macaron-python-package`.
```

Reply:
Addressed in https://github.com/oracle/macaron/pull/782/commits/874d7577e92969bc663cba56bd98b5fd435e3fa6